package.json

```diff
@@ -12,6 +12,9 @@
 
   "dependencies": {
     "babel": "^5.8.23",
+    "bluebird": "^2.10.2",
```

Comment:
So this makes it so that the random adb mock that I shell out to can use `mz` on nodejs `0.10`. We could specifically not support 0.10 though.